blueocean/pom.xml

```diff
@@ -115,6 +115,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>pipeline-build-step</artifactId>
+            <version>2.7-SNAPSHOT</version>
```

Comment:
This shouldn't be needed. Maybe its due to SNAPSHOT version?

Reply:
No idea. It makes sense to me that we shouldn't need it, but without it, kaboom. I'd say that there's some sort of intermediary in the lookup hierarchy between this pom and the root pom which is resetting pipeline-build-step back to the prod release. But debugging multi-pom projects is definitely something I suck at :(